test cases/common/125 shared module/module.c

```diff
@@ -9,54 +9,6 @@
   #endif
 #endif
 
-#ifdef _WIN32
```

Comment:
You are indeed not supposed to `link_with:` shared modules (on any platform); think of them as plugins. Linking to them at build time would defeat their purpose. They're supposed to be searched for and loaded with custom code (such as `g_module_open()`/`dlopen()`/flexdll) at runtime.